src/OrchardCore/OrchardCore/Environment/Shell/Builders/ShellContainerFactory.cs

```diff
@@ -126,11 +123,6 @@ public IServiceProvider CreateContainer(ShellSettings settings, ShellBlueprint b
                 });
             }
 
-            // Add a default configuration if none has been provided
-            var configuration = new ConfigurationBuilder().AddInMemoryCollection().Build();
-            moduleServiceCollection.TryAddSingleton(configuration);
-            tenantServiceCollection.TryAddSingleton(configuration);
-
```

Comment:
Useless, anyway it would need to be registered as `IConfiguration`.

    moduleServiceCollection.TryAddSingleton<IConfiguration>(configuration);
    tenantServiceCollection.TryAddSingleton<IConfiguration>(configuration);